IPs/RSKIP176.md

```diff
@@ -0,0 +1,145 @@
+# Fast BTC Bridge
+|RSKIP          |           |
+| :------------ |:-------------|
+|**Title**      |Fast BTC Bridge |
+|**Created**    |15-SEP-20 |
+|**Author**     |GM |
+|**Purpose**    |USa |
+|**Layer**      |Core |
+|**Complexity** |2 |
+|**Status**     |Draft |
+
+## Previous concepts
+
+#####Derived Federation
```

Comment:
space between #### and heading

Reply:
It seems that "Derived Federation" is not the right noun. 1) We not longer call it the "Federation". 2) You're referring to the address, not a new set of pegnatories. 
Maybe "Derived Powpeg Address" or, if you don't want to change all the document, "Derived Federation Address".

Reply:
Nothing should go before Abstract